docusaurus.config.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
           href: "https://github.com/amplication/amplication",
           label: "GitHub",
           position: "right",
-          activeSidebarClassName: "yuval",
+          activesidebarclassname: "yuval",
```

Comment:
This looks like a leftover from a test and should be removed

Reply:
Removed